content-security-policy/connect-src/connect-src-eventsource-blocked.sub.html

```diff
@@ -17,20 +17,15 @@
             log("violated-directive=" + e.violatedDirective);
         });
 
-        try {
-            var es = new EventSource("http://{{domains[www1]}}:{{ports[http][0]}}/content-security-policy/connect-src/resources/simple-event-stream");
-            // Firefox and Chrome don't throw an exception and takes some time to close async
-            if (es.readyState == EventSource.CONNECTING) {
-                setTimeout( function() {
-                    es.readyState != EventSource.CLOSED ? log("allowed") : log("blocked");
-                }, 1000);
-            } else if (es.readyState == EventSource.CLOSED) {
-                log("blocked");
-            } else {
-                log("allowed");
-            }
-        } catch (e) {
+        var es = new EventSource("http://{{domains[www1]}}:{{ports[http][0]}}/content-security-policy/connect-src/resources/simple-event-stream");
+        if (es.readyState == EventSource.CONNECTING) {
+            setTimeout( function() {
+                es.readyState != EventSource.CLOSED ? log("allowed") : log("blocked");
+            }, 1000);
```

Comment:
I think we can just end the test when `onerror` is fired? That would also save us a 1s timeout and a line in the lint.